code/game/objects/effects/landmarks.dm

```diff
@@ -286,3 +286,26 @@
 	ruin_landmarks -= src
 	ruin_template = null
 	. = ..()
+
+/obj/effect/landmark/stationary

```

Comment:
Oh, I added that for debug purposes. I needed something to not move with the shuttles so I could better understand the process. I can remove it if you want.